io.github.sigmasd.recordapps.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.sigmasd.recordapps
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # audio + grpahics
```

Comment:
you don't need this for audio.

Reply:
it doesn't work without it, for recording i have  to move the application audio to the virtual sink i created but that fail without that permission
basically these commands fails
```
pactl move-sink-input 2138 VirtualSink
ffmpeg -f pulse -i VirtualSink.monitor output.flac^C
```

Reply:
we are talking about `device=all`

Reply:
yes I mean device all, at runtime when i try to record without it I get permission error

Reply:
this is the error Jan 26 19:29:24 fedora pipewire-pulse[2472]: mod.protocol-pulse: client 0x55f5e0f8f9f0 [pactl]: ERROR command:67 (MOVE_SINK_INPUT) tag:2 error:1 (Permission denied)

Reply:
which device is it missing ?

Reply:
I don't know i tried all the available one (dri,shm,..) they didn't work even tough it set them all, it seems like a limitation in this api that should be improved

Reply:
Talking about `/dev/` device node here.

Reply:
Do you have an idea how do I proceed further  ?
You can reproduce with this manifest
```yml
app-id: io.github.sigmasd.recordapps
runtime: org.gnome.Platform
runtime-version: "47"
sdk: org.gnome.Sdk
command: run.sh
finish-args:
  - --socket=pulseaudio
  - --device=all # comment  out to trigger error
modules:
  - name: recordapps
    buildsystem: simple
    build-commands:
      - install -D run.sh /app/bin/run.sh
      - chmod +x /app/bin/run.sh
    sources:
      - type: script
        dest-filename: run.sh
        commands:
          - pactl move-sink-input $(pactl list short sink-inputs | awk '{print $1}' | head -n 1) $(pactl list short sinks | awk '{print $1}' | head -n 1)
```